TdsModel.h

```diff
@@ -297,6 +297,9 @@ class TdsModel
   void getSectionFlow(int sectionIndex, double &inflow, double &outflow);
   double getSectionPressure(int sectionIndex);
 
+  void calcNoiseSample(NoiseSource* s, double ampThreshold);
```

Comment:
Why does this need to be public?

Reply:
This was to be able to use the noise source function already implemented, but it turns out that this function is too complex for what we need so I found it simpler to just generate random numbers.